metricbeat/docs/modules/aerospike/namespace.asciidoc

```diff
@@ -7,6 +7,7 @@ This file is generated! See scripts/mage/docs_collector.go
 
 include::../../../module/aerospike/namespace/_meta/docs.asciidoc[]
 
+This is a default metricset. If its host module is unconfigured, it will be enabled by default.
```

Comment:
If it's not too late, I'd replace this text as follows. The reason is that the original is a bit pronoun heavy, which can make it harder to read.

```suggestion
This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```